filament/backend/src/vulkan/VulkanDriver.h

```diff
@@ -153,6 +153,7 @@ class VulkanDriver final : public DriverBase {
     VulkanSamplerGroup* mSamplerBindings[VulkanPipelineCache::SAMPLER_BINDING_COUNT] = {};
     VkDebugReportCallbackEXT mDebugCallback = VK_NULL_HANDLE;
     VkDebugUtilsMessengerEXT mDebugMessenger = VK_NULL_HANDLE;
+    FeatureLevel mFeatureLevel = FeatureLevel::FEATURE_LEVEL_1;
```

Comment:
NBD but I would not have bothered adding state here since we already persist the entire device caps and determining the answer to the query is a one liner.

Reply:
ok, Engine is going to cache the result anyways. so yeah, I can change that.

Reply:
done